docs/source/models/causes/malaria/index.rst

```diff
@@ -0,0 +1,289 @@
+	  - Disability weight
+	* - Mild
+	  - Mild malaria	Has a low fever and mild discomfort but no difficulty with daily activities.	
+	  - 0.006 (0.002–0.012)
+	* - Moderate
+	  - Has a fever and aches and feels weak, which causes some difficulty with daily activities. 
+	  - 0.051 (0.032–0.074)
+	* - Severe
+	  - Has a high fever and pain and feels very weak, which causes great difficulty with daily activities. 
+	  - 0.133 (0.088–0.19)
+
+
+GBD Hierarchy
+-------------
+
+.. image:: malaria_cause_hierarchy.svg
+
+Cause Model Diagram
+-------------------
+
+.. image:: malaria_cause_model.svg
+
+
+S: **S**\ usceptible to malaria
+
+I: **I**\ nfected and currently experiencing malaria
+
+
+Model Assumptions and Limitations
+---------------------------------
+
+Malaria has been modeled extensively by other teams. For this model, 
+we will not be including any information on vectors and so it is not 
+appropriate for interventions targetting malaria prevention or treatment 
+directly. 
+
+There is evidence that people living in malaria endemic areas do gain immunity over 
+their lifetimes. We assume this is represented in prevalence and incidence rates from 
+GBD. We do not include in this model any gains in malaria resistance from prior exposure. 
+
+.. todo::
+
+   Continue to add to this section as needed 
+
+Data Description
+----------------
+
+.. list-table:: State Definitions
+	:widths: 5 10 10
+	:header-rows: 1
+	
+	* - State
+	  - State name
+	  - Definition
+	* - S
+	  - **S**\ usceptible
+	  - Simulant does not currently have malaria disease
+	* - I
+	  - **I**\ nfected
+	  - Simulant currently has malaria
+
+.. list-table:: State Data
+	:widths: 5 10 10 20
+	:header-rows: 1
+	
+	* - State
+	  - Measure
+	  - Value
+	  - Notes
+	* - S
+	  - prevalence
+	  - 1-prevalence_I
+	  - 
+	* - S
+	  - birth prevalence
+	  - 1 
+	  - 
+	* - S
+	  - emr
+	  - 0
+	  -
+	* - S
+	  - disability weight
+	  - 0
+	  -
+	* - I
+	  - prevalence
+	  - **For early neonatal age group:** (birth_prevalence_I + prevalence_345)/2. **For all other age groups:** prevalence_345
+	  - 
+	* - I
+	  - birth prevalence
+	  - 0 
+	  - 
+	* - I
+	  - excess mortality rate
+	  - :math:`\frac{\text{deaths_c345}}{\text{population} \,\times\, \text{prevalence_345}}`
+	  - 
+	* - I
+	  - disability weight
+	  - :math:`\displaystyle{\sum_{s\in \text{sequelae_malaria}}} \scriptstyle{\text{disability_weight}_s \,\times\, \text{prevalence}_s}`
+	  - Malaria sequelae are: 121, 122, 123
+	* - All
+	  - cause-specific mortality rate
+	  - :math:`\frac{\text{deaths_c345}}{\text{population}}`
+	  -
+
+.. list-table:: Transition Data
+	:widths: 10 10 10 10 10
+	:header-rows: 1
+	
+	* - Transition
+	  - Source State
+	  - Sink State
+	  - Value
+	  - Notes
+	* - i
+	  - S
+	  - I
+	  - :math:`\frac{\text{incidence_rate_c345}}{1-\text{prevalence_I}}`
+	  - Incidence in GBD are estimated for the total population. Here we transform incidence to be a rate within the susceptible population.
```

Comment:
```suggestion
	  - Above is equivalent to "load standard data" Vivarium public health function ("suscpetible-population incidence rate). Incidence in GBD are estimated for the total population. Here we transform incidence to be a rate within the susceptible population.
```

The engineers have discussed that it would be helpful if we indicate whether or not they should use the "load standard data" functions for these kind of things. Here, load standard data will directly return what's included in this table, so just adding an additional note about that